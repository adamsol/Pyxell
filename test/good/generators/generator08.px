
func codes(s): Int*Char*Int... def
    for c, i in s, 0... do
        yield i, c, c.code

print [...codes("a*")]
