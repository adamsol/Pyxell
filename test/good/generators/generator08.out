[(0, 'a', 97), (1, '*', 42)]
